src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -52,8 +54,14 @@
      */
     private $reflector;
 
+    /**
+     * @var Parser
+     */
+    private $parser;
+
     protected function __construct()
     {
+        $this->parser = (new Configuration())->phpParser();
```

Comment:
This is probably the offending memory leak: each function has a new parser.

I can probably revert the changes in here, as it's not really needed in the constructor, and can probably even be static (only used in `setBodyFromClosure` and `setBodyFromString`)

Reply:
Both methods are removed in https://github.com/Roave/BetterReflection/pull/339